pkg/3scale/amp/component/apicast.go

```diff
@@ -90,8 +95,9 @@ func (apicast *Apicast) StagingDeploymentConfig() *appsv1.DeploymentConfig {
 	return &appsv1.DeploymentConfig{
 		TypeMeta: metav1.TypeMeta{APIVersion: "apps.openshift.io/v1", Kind: "DeploymentConfig"},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   ApicastStagingName,
-			Labels: apicast.Options.CommonStagingLabels,
+			Name:        ApicastStagingName,
```

Comment:
I see the annotations have been set in the `DeploymentConfig` annotations, and not in the PodTemplateSpec annotations of the DeploymentConfig. Is this intentional? The difference is that if the annotation is changed here there will be no redeployment of the pods of the DeploymentConfig, whereas if we put them in the PodTemplateSpec the pods would be redeployed

Reply:
Yes it is intentional. Those annotations are for operator tracking purposes. Adding/updating/removing those annotations should not be the only reason to respin a deployment. An also, there is not a good reason to have them in the pod. IMO, this is data to operate at DC level.

Reply:
:+1: